diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_cs.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_cs.c
index 0a1ab92..98b6572 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_cs.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_cs.c
@@ -134,7 +134,7 @@ static int amdgpu_cs_user_fence_chunk(struct amdgpu_cs_parser *p,
 	uint32_t handle;
 
 	handle = fence_data->handle;
-	gobj = drm_gem_object_lookup(p->filp, data->handle);
+	gobj = drm_gem_object_lookup(p->filp, fence_data->handle);
 	if (gobj == NULL)
 		return -EINVAL;
 
